server/assets/javascripts/accessibility.js

```diff
@@ -0,0 +1,8 @@
+//File used strictly for updating HTML for accessibility purposes
+function loginAccessibility(){
```

Comment:
Not sure if it's worth putting this in it's own file? Could just be in login.js?